README.rst

```diff
@@ -28,133 +28,43 @@ Nengo: Large-scale brain modelling in Python
 Installation
 ============
 
-To install Nengo, use::
+To install Nengo::
```

Comment:
How do you feel about a link to the dev install installation instructions here? Would that address your concern?